src/styles/felipec.css

```diff
@@ -7,26 +7,26 @@
 */
 
 .hljs {
-  color: #dddde1;
-  background: #1e1e22;
+  color: #dedde4;
+  background-color: #1d1c21;
```

Comment:
Is there a reason for the attribute change?

Reply:
No reason. I used `background` initially because `check-theme` failed otherwise, but I fixed that on 0e3735ac.